index.html

```diff
@@ -711,6 +711,58 @@ <h3>Hashlinks</h3>
 
     </section>
 
+    <section>
+      <h2>Progressive Trust</h2>
```

Comment:
What is "Progressive Trust"?  How do the things below enable it?

Reply:
I have added some text (Thanks @dlongley) in [0e91eea](https://github.com/w3c/vc-imp-guide/pull/14/commits/0e91eeafe33a5a1ed8ba2dea4c26dbfb5f6d8367) which I hope answers your questions.